lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/KBVAnswerStorageService.java

```diff
@@ -0,0 +1,57 @@
+package uk.gov.di.ipv.cri.kbv.api.service;
+
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbEnhancedClient;
+import software.amazon.awssdk.enhanced.dynamodb.DynamoDbTable;
+import software.amazon.awssdk.enhanced.dynamodb.TableSchema;
+import software.amazon.awssdk.enhanced.dynamodb.model.BatchWriteItemEnhancedRequest;
+import software.amazon.awssdk.enhanced.dynamodb.model.WriteBatch;
+import uk.gov.di.ipv.cri.common.library.persistence.DynamoDbEnhancedClientFactory;
+import uk.gov.di.ipv.cri.common.library.service.ConfigurationService;
+import uk.gov.di.ipv.cri.kbv.api.domain.KBVAnswerItem;
+import uk.gov.di.ipv.cri.kbv.api.domain.QuestionsResponse;
+import uk.gov.di.ipv.cri.kbv.api.util.KBVAnswerStorageMapper;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class KBVAnswerStorageService {
+    private final DynamoDbEnhancedClient enhancedClient;
+    private final KBVAnswerStorageMapper kbvAnswerStorageMapper;
+    private final String tableName;
+
+    public KBVAnswerStorageService(
+            DynamoDbEnhancedClient enhancedClient,
```

Comment:
Remove the dependency on `DynamoDbEnhancedClient`
Make this class dependent on the DataStore and you can use the new `createItems` method
Then there will be no need for any references to the aws sdk